apps/src/gamelab/spritelab/spriteUtils.js

```diff
@@ -75,6 +75,20 @@ export function getBehaviorsForAnimation(animation) {
   return numBehaviors;
 }
 
+/**
+ * @param {number} spriteId
+ * @return {number} Number of behaviors associated with the specified sprite
+ */
+export function getBehaviorsForSpriteId(spriteId) {
```

Comment:
Nit: I would expect 'getBehaviorsForSpriteId' to return the behaviors, not the number of behaviors.